### DISCUSSION TITLE
Thank you for your contributions in February 2024

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11991] [Community]
We're grateful for all the wonderful contributions we received in February in Kolibri, Studio, and KDS repositories, for both backend and frontend. We merged accessibility features, development tooling improvements, dependencies upgrades, bug fixes, new design system features, a new frontend testing library, and maintenance refactorings such as routing improvements.

Thank you @muditchoudhary @ShivangRawat30 @ThEditor @GarvitSinghal47 @KshitijThareja @nick2432 @thesujai @FidalMathew @iskipu @EshaanAgg @nikkuAg @Mohamedkhaled81 and also all contributors who have work in progress or waiting for review.

As usual, we'll mention all of you in our next [Community Newsletter](https://learningequalitynewsletter.getresponsepages.com/).

Warm regards

**Comment by radinamatic** [comment]
Thank you all for your contributions, very much appreciated! 👏🏽 💯 ❤️
